### PR TITLE
fix(Mint): No redirect to Tokens screen when minting a token

### LINKED ISSUE #11731: Community -> Minting: No redirect to Tokens screen when minting a token
### Description

I am stuck on this screen even the transaction for minting is completed a while ago. I think we did the redirect to Tokens page where we show `Minting` status for a token

Screenshot 2023-08-01 at 12 56 49

### PR DESCRIPTION
**Pull request by noeliaSD**
Fixes #11731

### What does the PR do

- It updates `MintTokensSettingsPanel.qml` to properly behaves when reseting navigation.

### Affected areas

Community Settings / Tokens

### CODE REVIEW

**Comment by noeliaSD** [comment]
@anastasiyaig already tested it!

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml [hunk 1]
```diff
@@ -140,71 +135,65 @@ StackView {
...
 
-    initialItem: mintedTokensViewComponent
-
-    Component {
-        id: tokenObjectComponent
-
-        TokenObject {}
-    }
+    initialItem: SettingsPage {
+        implicitWidth: 0
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:139]: why `0` here?

**Review by caybro — APPROVED** [review]
Otherwise looks good

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml (hunk 1 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:139]: I've just moved this code from a component to a direct assignment. Going to check that..

Comment (micieslak) [reply ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:139]: On Qt 5.15.2 (which is still used for deployment on linux from what I know) there is binding loop on implicitWidth which can be fixed by setting it to `0` explicitly. It's safe bc implicitWidth is not used here anyway.

**Review by micieslak — APPROVED** [review]
Looks nice!

**Pull request merged**